### PR TITLE
zombie powder is delayed now instead of isntant

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
zombie powder now gives you confusion and drowsyness for 10 seconds
zombie powder now makes you fakedeath after 10 seconds and cures when you lose the reagent although i might have fucked it up but it should cure
closes #46803

## Why It's Good For The Game

i find it better than the solution acapitala took, changing it into not zombie powder at all
also cobbyman said that it should be like this

## Changelog
:cl:
balance: zombie powder now activates on the 5th cycle, around 10 seconds
:cl:

### CODE REVIEW

**Comment by AcapitalA** [comment]
Instead of complaining about points I already explained myself the madlad actually makes his own PR and does it his way. Absolute chad.

No seriously, consider merging this one over mine if you wanna.

**Comment by ExcessiveUseOfCobblestone** [comment]
closes the other one...?

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -163,16 +163,19 @@
...
 
 /datum/reagent/toxin/zombiepowder/on_mob_life(mob/living/carbon/M)
 	M.adjustOxyLoss(0.5*REM, 0)
+	switch(current_cycle)
+		if(1 to 5)
+			M.confused += 1
+			M.drowsyness += 1
+		if(5 to INFINITY)
+			M.fakedeath(type)
+			. = 1
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: ?

**Comment by Shadowflame909** [comment]
Copying the comment I wrote there since it applys more here.

Reflecting on it more. I don't really like that solution. Also even with the delay, what would be the difference between it and chloral hydrate?

Either way you're still dragging a body to a remote location with your target screaming bloody murder and running before the KO.

Except someone observing you will think you're a ling instead of a traitor.

You put in a bunch of effort for a KO chem of a higher tier. That being ZP.

If you want it even more out of reach. Just do a full on romeo and Juliet reference and have it require a priest giving you holy water.

But yeah. Any nerf otherwise just makes it weaker then a chem that requires less work. Or on the same level.

**Comment by Krysonism** [comment]
>But yeah. Any nerf otherwise just makes it weaker then a chem that requires less work. Or on the same level.

Sleeping on the 5th cycle is insanely strong. Way stronger than chloral. That is even without taking the confusion into account.

**Comment by davethwave** [comment]
still better than as soon as injected like before

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: maybe go from 6 to INFINITY instead of 5 to INFINITY?

Comment (ATH1909) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: should it mute for the first 10 seconds, or would that be too strong?

**Comment by Shadowflame909** [comment]
Gonna need the break down on that. Is 1 cycle every 10 ticks? And since a tick is 1 - 2 seconds depending on time dialation. Would the KO happen  under 1 to 2 minutes if the lag is bad.

If my math be right. Thats bad gamer

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (Fikou) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: this was cargo cult

**Comment by Fikou** [comment]
> Gonna need the break down on that. Is 1 cycle every 10 ticks? And since a tick is 1 - 2 seconds depending on time dialation. Would the KO happen under 1 to 2 minutes if the lag is bad.

1 cycle is every tick im pretty sure

**Comment by ExcessiveUseOfCobblestone** [comment]
5 is hardly a delay tbh

**Comment by Shadowflame909** [comment]
Pretty sure chloral's 15

it's a 2 step program where if zombie powder is 10 then zombie powder just isn't reasonable to make anymore

**Comment by Ty-the-Smonk** [comment]
Should probably mute the person after 2 cycles

**Comment by PKPenguin321** [comment]
@ExcessiveUseOfCobblestone chloral is also way easier to make and way less useful, so

**Comment by ExcessiveUseOfCobblestone** [comment]
> Pretty sure chloral's 15
> 
> it's a 2 step program where if zombie powder is 10 then zombie powder just isn't reasonable to make anymore

we're both wrong it's cycle 50.

> @ExcessiveUseOfCobblestone chloral is also way easier to make and way less useful, so

ZP isn't hard to make, you just main the wrong department.

**Comment by PKPenguin321** [comment]
well if you have an emag sure (not seen in chemistry in majority of rounds), but even then it's like 1u for 15u of input, and besides chloral is still way less useful

**Comment by ExcessiveUseOfCobblestone** [comment]
like i said, wrong department. You can reliably make ZP everyround if you wanted to 😏

**Comment by PKPenguin321** [comment]
sure you can 🙄

**Comment by ExcessiveUseOfCobblestone** [comment]
it doesn't have to have as strong of a delay as chloral, but it needs to be something that provides a window of reaction (again chloral does 50 so you have a lot of wiggles between 5 and 50).

> sure you can 🙄

if you can get copper you can, that's ironically your limiter in the reaction as well as the container volume you're using since the input:output is fairly wild (not that it matters too much for the department that can churn it out cough cough).

**Comment by Krysonism** [comment]
>we're both wrong it's cycle 50.

No it is not. Have you ever used chloral? It obviously does not take 100 seconds to put someone to sleep with it.

It's 10 or 11 cycles or about 20 seconds.

**Comment by Shadowflame909** [comment]
Cobby gets spooky when dealing with nerf PRS

Think of the quality of the playerbase. No ones expecting high rp level of office worker balance.

The concoction of instant  knock outs is peak /tg/ effort for the grand reward

Also how are they making ZP so easy as a non antag

**Comment by 81Denton** [comment]
@ExcessiveUseOfCobblestone 
```
		if(10 to 50)
			M.Sleeping(40, 0)
			. = 1
```
Cycles 51+ are just sleeping + toxloss

**Comment by PKPenguin321** [comment]
>Also how are they making ZP so easy as a non antag

With just copper apparently since you can supposedly get carpotoxin for free without an emag or (2u of zp per) space carp or departmental cooperation 🙄

**Comment by Krysonism** [comment]
Well you can grow koibeans in the garden. It's pretty easy if you don't need much. You only need a few units to kill someone.

**Comment by PKPenguin321** [comment]
because that's so easy and fast right? that's absolutely not 0 effort like cobby makes it sound, and you still need to actually acquire the seeds which can be easier said than done in rounds with a botanist (unless bean seeds start in the garden on some maps, in which case bad map)

**Comment by Shadowflame909** [comment]
Honestly I've never heard of someone using carpo to kill

But the mixture rate is borderline unusuable.

Its an instant KO in exchange for being draining of multiple limited resources

Well multiple until someone made morphine creatable via chemistry.

2u for killing a spess fish/using mutagen and saltpetre on a botanical plant and looting some morphine and copper

Considering we already have end game chems balanced by a scavenger hunt of a recipe. Making it involve more deparmental cooperation is the solution, I feel.

**Comment by Krysonism** [comment]
How fast depends on how fast the miners bring in uranium. You can do it faster if you ask the chemists for mutagen.

It is not totally effortless of course but it is essentially a one hit and win weapon.

**Comment by Shadowflame909** [comment]
What if it involved stable uranium gel or strange reagent in its creation.

Something like Ghoul Powder (With a changed recipe maybe Neurotoxin replacing ZP as an ingredient) , Strange Reagent, Stable Uranium Gel

**Comment by AdamElTablawy** [comment]
just make it ingest only and leave it as is rather than making it chloral but mildly cooler.

that or increase the yield amount to be more in line with chloral so that it's not a tiny amount since it's not an instant KO anymore

**Comment by ExcessiveUseOfCobblestone** [comment]
growing plants are easy, you only need 1 koibean for the reagent then you can move it to (the best reagent farming plant in the game).

Naturally it's not as easy as getting the elusive 6tc item but from a core game perspective it's certainly not hard to get it compared to other items of equal value.

**Comment by ExcessiveUseOfCobblestone** [comment]
> @ExcessiveUseOfCobblestone
> 
> ```
> 		if(10 to 50)
> 			M.Sleeping(40, 0)
> 			. = 1
> ```
> 
> Cycles 51+ are just sleeping + toxloss

i can read code, h-h-h-honest.

**Comment by Shadowflame909** [comment]
If it was ingest only, and thus a melee stun like every stun left in the game. 

Would that be okay?

**Comment by ExcessiveUseOfCobblestone** [comment]
why not do both? If injected it works on a delay but if ingested it works instantly.

If you can force someone to take a patch you've already won the fight anyways.

**Comment by Shadowflame909** [comment]
Honestly, we have a neat mechanic where you can stealthily crush pills into beakers/glassware that definitely needs more use.

If someone's not wearing any form of mouthwear, you can also smother people from behind with a damp rag.

**Comment by ExcessiveUseOfCobblestone** [comment]
i'm fine with that until I see it abused in practice 😏

**Comment by Fikou** [comment]
shadowflame's version is pretty based

**Comment by 81Denton** [comment]
Closing this in favor of #46931

**Pull request closed without merging**